book/website/community-handbook/infrastructure/infrastructure-contributors.md

```diff
@@ -0,0 +1,42 @@
+# Contributors
+
+Contributors are published in the book's {ref}`Record of Contributions<contributors-record>`.
```

Comment:
```suggestion
Keeping track of contributions of all types is important: both code and non-code contributors.
Within The Turing Way, contributors are published in the book's {ref}`Record of Contributions<contributors-record>`, as well as in our [Contributors Table](https://twitter.com/ZeynepEngin/status/1661382725687824388) on Github. 
```

Reply:
I think that is the wrong link 😄.

Were you thinking of the all contributors table in the README or `contributors.md`?
The contributors record page combines both (plus it has some addition information itself).

Reply:
Maybe the link out to "how to tag contributors" should go to [here](https://the-turing-way.netlify.app/community-handbook/acknowledgement.html)?